charts/acme/tests/defaults_test.yaml

```diff
@@ -0,0 +1,23 @@
+suite: default tests
+values:
+  - ../values.yaml
+  - secrets_default_values.yaml
```

Comment:
```suggestion
```

The "default test" should not provide any (helm) value at all as it is expected to simulate the `helm template ./` command.

I believe it *might* break the test and could require more changes (sorry 😅)

Reply:
Got your point
I split the unit test in two files: custom test and default test. Let me know if that looks good to you.

I actually took example of the accountapp chart to redact theses tests, which doesn't follow this structure. If you create an issue I can fix it !